Sources/ComposableArchitecture/Reducer/AnyReducer/AnyReducerSignpost.swift

```diff
@@ -1,4 +1,9 @@
+#if canImport(OSLog)
+  #if canImport(OpenCombine)
```

Comment:
Actually, is there a case where `canImport(OSLog)` will be true but `canImport(Combine` will be false?

Reply:
Great question, I'm not sure. I don't know what OSLog availability outside of the Apple ecosystems is. This was likely a result of find and replace. Let me take a look at this file, but it likely should be Apple platform _only_ since none of the signposting stuff exists outside of Apple stuff so I can probably reduce this diff.